src/explorer/fileListing/onDraw.js

```diff
@@ -6,11 +6,7 @@ export function onDraw(explorer) {
     this.table
       .select('tbody')
       .selectAll('tr')
-      .filter(
-        f =>
-          f[explorer.config.labelColumn] ===
-          explorer.current[explorer.config.labelColumn]
-      )
+      .filter(f => f.fileID === explorer.current.fileID)
```

Comment:
could also do `.classed('selected', f => f.fileID === explorer.current.fileID)` because i like to nitpick

Reply:
yeah, that would be cleaner. I'll commit the update.